examples/hello.py

```diff
@@ -0,0 +1,19 @@
+from toil.common import Toil
```

Comment:
It looks like this file is already at: https://github.com/DataBiosphere/toil/blob/d23f7ec46d2006c136a2a5b4e57eadfb44a606b7/src/toil/test/docs/scripts/tutorial_arguments.py

Can we pull from there?  I'd rather not have two of the same script.

Reply:
Hm, I didn't know that was there. I can make this a symlink probably.